plugins/tileacq.py

```diff
@@ -568,6 +593,53 @@ def _check_fov(self, das, sfov):
             sfov = asfov
         return sfov
 
+    MEMPP = 2e-8  # memory per pixel in GB, found empirically
+    def _memory_check(self, _=None):
+        """
+        Makes an estimate for the amount of memory that will be consumed during
+        stitching and compares it to the available memory on the computer.
+        Displays a warning if memory exceeds available memory.
+        returns (bool): True (memory is sufficient), False (memory will be exhausted)
+        """
+
+        if self.stitch.value:
+            # Number of pixels for acquisition
+            pxs = 0
+            for s in self._get_acq_streams():
+                if hasattr(s, 'emtResolution'):
+                    pxs += s.emtResolution.value[0] * s.emtResolution.value[1]
+                elif hasattr(s, 'repetition'):
+                    pxs += s.repetition.value[0] * s.repetition.value[1]
+                else:
+                    logging.warning("Resolution of stream %s cannot be determined." % s)
+            pxs = pxs * self.nx.value * self.ny.value
+
+            # Memory calculation
+            mem_est = pxs * self.MEMPP
+            mem_computer = psutil.virtual_memory().total / 1024 ** 3  # in GB
+            # Assume computer is using 2 GB RAM for odemis and other programs
+            if mem_est < mem_computer - 2:
+                mem_check = True
```

Comment:
`mem_check` is not a very clear name for a boolean. I'd propose `mem_sufficient`.

Also, as there is only a boolean in the `if`, you can rewrite it as `mem_sufficient = mem_est < mem_computer - 2`.